solr/core/src/java/org/apache/solr/handler/designer/SchemaDesignerConfigSetHelper.java

```diff
@@ -125,46 +119,22 @@ class SchemaDesignerConfigSetHelper implements SchemaDesignerConstants {
   }
 
   @SuppressWarnings("unchecked")
-  Map<String, Object> analyzeField(String configSet, String fieldName, String fieldText)
+  NamedList<Object> analyzeField(String configSet, String fieldName, String fieldText)
```

Comment:
returning NamedList now (more Solr native); didn't need the Map conversion or JSON stuff or InputStream low level stuff

Reply:
Good stuff!